nestkernel/connection_manager.cpp

```diff
@@ -1122,7 +1116,13 @@ nest::ConnectionManager::get_connections( ArrayDatum& connectome,
 #ifdef _OPENMP
 #pragma omp critical( get_connections )
 #endif
-        connectome.append_move( conns_in_thread );
+        {
+          while ( !conns_in_thread.empty() )
```

Comment:
@alexeyshusharin This code repeats two more times below, so I would suggest to refactor it into a method.

Two more details: 
In newly written NEST code, logical operators (`!`, `&&`, ÌI`) should be written out as `not`, `and`, `or` for greater clarity.

I am also wondering if `std::copy()` wouldn't be faster and more expressive, although it would come at the price of temporarily having two copies of the connectivity data. But since the doubling is per-thread only, I think that would not be too bad.